import_export/widgets.py

```diff
@@ -128,6 +130,32 @@ def render(self, value, obj=None):
             )
         return value
 
+    def clean_numeric_separators(self, value):
+        """
+        Replace language-specific numeric separators with the versions parsed
+        by Python’s number classes: “.” for the decimal separator and no
+        thousand separators.
+        """
+        if not isinstance(value, str):
+            return value
+        return value.translate(get_number_translation(get_language()))
+
+
+@lru_cache
+def get_number_translation(language):
+    decimal_separator = get_format("DECIMAL_SEPARATOR", language)
+    thousand_separator = get_format("THOUSAND_SEPARATOR", language)
+    table = {
+        # Must be exactly one character. Django technically allows other
+        # lengths but all built-in languages use one character.
+        decimal_separator: ".",
+    }
+    # Some languages, like zh-hans, use an empty thousand separator, which is
```

Comment:
I think the unit tests are arranged incorrectly, as you have this test for the "fr" locale not "zh-hans", I might be missing something.

Reply:
Ah, I can see why you might think that, but it’s not like that. The zh-hans tests use `1234` - no separator between 1 and 2. The French ones use `\xa0` - a non-breaking space, per:

https://github.com/django/django/blob/a16f13a8661297eda12c4177bb01fa2e5b5ccc56/django/conf/locale/fr/formats.py#L26